packages/lib/shared/utils/numbers.ts

```diff
@@ -18,6 +18,28 @@ BigInt.prototype.toJSON = function () {
 export const MAX_BIGINT = BigInt(MAX_UINT256)
 export const MAX_BIGNUMBER = bn(MAX_UINT256)
 
+// Dash symbol used for zero balances and empty values
+export const ZERO_VALUE_DASH = '-'
+
+// Display labels for small amounts
+export const SMALL_AMOUNT_LABEL = '<0.001'
+export const SMALL_PERCENTAGE_LABEL = '<0.01%'
+
+// Thresholds for display formatting
+export const AMOUNT_LOWER_THRESHOLD = 0.001
+export const PERCENTAGE_LOWER_THRESHOLD = 0.0001
+export const USD_LOWER_THRESHOLD = 0.009
+
+// APR display thresholds
+export const APR_UPPER_THRESHOLD = 1_000_000
+export const APR_LOWER_THRESHOLD = 0.0000001
+
+// BigNumber processing threshold
+export const BN_LOWER_THRESHOLD = 0.000001
+
+// Fiat formatting threshold
+export const FIAT_CENTS_THRESHOLD = '100000'
+
```

Comment:
please keep the original comments and only change what you are adding here

Reply:
done, thx